frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/CreateContractor/ContractorDetailTable.js

```diff
@@ -0,0 +1,270 @@
+import React, { Fragment, useState } from 'react'
+import { AddIcon, DeleteIcon, RemoveIcon,Dropdown,TextInput, CardLabelError,DatePicker } from '@egovernments/digit-ui-react-components'
+
+const WORKSContractorTable = ({ t, register, errors, rows, setRows,Controller,control }) => {
+
+    // const initialState = [
```

Comment:
@nipunarora-eGov  & @burhanj-eGov remove all the console.logs , debuggers and unnecessary codes.
Format the files